README.md

```diff
@@ -1,11 +1,17 @@
 [![Build Status](https://travis-ci.org/OCA/margin-analysis.svg?branch=8.0)](https://travis-ci.org/OCA/margin-analysis)
 [![Coverage Status](https://img.shields.io/coveralls/OCA/margin-analysis.svg)](https://coveralls.io/r/OCA/margin-analysis?branch=8.0)
 
-Margin Analysis
-===============
+Margin Analysis / Financial controlling
+=======================================
 
 This project aim to deal with modules related to manage Financial controlling and costing in a generic way. You'll find modules that:
 
  - Compute cost based on BoM
  - Compute standard and historical margin
+ - Price historization for reporting & BI
  - ...
+
+
+# Translation state
```

Comment:
Use a lower header size with `------------------` like the template https://raw.githubusercontent.com/OCA/maintainer-quality-tools/master/sample_files/README.md